core/src/apps/solana/sign_tx.py

```
@@ -62,6 +62,10 @@ async def sign_tx(
     # verify fee payer
     if not multiple_signers:
         if fee_payer.get() != signer_pub_key_bytes:
+            if __debug__:
+                print(
+                    f"Invalid signer used: {PublicKey(fee_payer.get())} != {PublicKey(signer_pub_key_bytes)}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)